orchagent/vnetorch.cpp

```diff
@@ -793,6 +892,70 @@ bool VNetBitmapObject::addTunnelRoute(IpPrefix& ipPrefix, tunnelEndpoint& endp)
         throw std::runtime_error("VNet route creation failed");
     }
 
+    tunnelRouteInfo.vni = endp.vni == 0 ? getVni() : endp.vni;
+    tunnelRouteInfo.mac = mac;
+    tunnelRouteMap_.emplace(ipPrefix, tunnelRouteInfo);
```

Comment:
Do you want to add a DEBUG log here for the create? Please ignore if it is logged elsewhere in the flow.